### PR TITLE
Add a sort-by-value checkbox.

### PR DESCRIPTION
**Pull request by bwinton**
As requested by me, and described by @chutten. 🙂

### CODE REVIEW

File: new-pipeline/src/dist.js [hunk 1]
```diff
@@ -914,6 +918,28 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim, in
     if (histogramsList.length === 1 && histogramsList[0].histograms.length ===
       1) { // Only show one set of axes
       var histogram = histogramsList[0].histograms[0];
+      if (sortByValue) {
+        let sorting = [];
+        for (let index = 0; index < histogram.buckets.length; index++) {
+          sorting.push([histogram.values[index], histogram.buckets[index]]);
+        }
+        sorting.sort((a,b) => {
```
Comment (chutten) [inline new-pipeline/src/dist.js]:
```suggestion
        sorting.sort((a, b) => {
```

File: new-pipeline/src/dist.js [hunk 2]
```diff
@@ -914,6 +918,28 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim, in
...
+          rv = b[0] - a[0];
```
Comment (chutten) [inline new-pipeline/src/dist.js]: `rv` needs a `let` or `var` or something

File: new-pipeline/src/dist.js [hunk 3]
```diff
@@ -914,6 +918,28 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim, in
...
+          if (rv != 0) {
+            return rv;
+          }
+          if (a[1] < b[1]) {
+            return -1;
+          }
+          if (a[1] > b[1]) {
+            return 1;
+          }
+          return 0;
```
Comment (chutten) [inline new-pipeline/src/dist.js]: If you think it's not too difficult to read/maintain we could condense to:

```js
sorting.sort((a, b) => b[0] - a[0] || a[1].localeCompare(b[1]));
```

**Review by chutten — APPROVED** [review]
Some nits, but it looks good!

**Comment by chutten** [comment]
Now that I've given a few more moments' thought to it, maybe we should limit this to histograms that are of categorical kind. A linear or exponential histogram should probably _never_ be sorted by value.

File: new-pipeline/src/dist.js (hunk 1 above)
Comment (bwinton) [reply new-pipeline/src/dist.js]: Fixed (kinda).

File: new-pipeline/src/dist.js (hunk 2 above)
Comment (bwinton) [reply new-pipeline/src/dist.js]: Fixed (kinda).

File: new-pipeline/src/dist.js (hunk 3 above)
Comment (bwinton) [reply new-pipeline/src/dist.js]: Changed to this, which removes the previous two nits. 🙂

**Comment by bwinton** [comment]
Something like this latest one?

**Review by chutten — APPROVED** [review]

**Comment by chutten** [comment]
Alrighty, it should be up within the next half-hour or so. May need to force-refresh to get through the layers of caching.

Fingers crossed that not everything explodes!

**Pull request merged**